src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -2387,7 +2411,90 @@ components:
       type: string
       enum:
         - ResourceBadge
-        - NonFungibleResourceBadge
+        - NonFungibleBadge
+
+    AccountDefaultDepositRule:
+      type: string
+      description: Default deposit rule set to an account.
+      enum:
+        - Accept
+        - Reject
+        - AllowExisting
+
+    AccountDepositPreValidationDecidingFactors:
+      description: Deciding factors used to calculate response.
+      type: object
+      required:
+        - default_deposit_rule
+      properties:
+        is_badge_authorized_depositor:
+          description: Whether the input badge belongs to the account's set of authorized depositors. This field will only be present if any badge was passed in the request.
+          type: boolean
+          nullable: true
```

Comment:
I think it's enough to not enlist this type in `required`?

Reply:
it defaulted to false without it. I think it handles differently simple types.